src/User/Register/RegisterFormBuilder.php

```diff
@@ -54,6 +54,18 @@ class RegisterFormBuilder extends FormBuilder
         'activation_code',
         'last_activity_at'*/
     ];
+    
+    /**
+     * The form actions.
+     *
+     * @var array
+     */
+    protected $actions = [
+        'register' => [
```

Comment:
And really a pretty sure this can be blue and remove button => blue just to clean it up a smidge